### DISCUSSION TITLE
Automatic Rank Assigned (Core>Groups & Forum Module)

### DISCUSSION THREAD

**Discussion by RawrMaoster** [NamelessMC/Nameless#2343] [Ideas]
Automatic Rank Assigned

It would be a neat feature if i and anyone else could have roles (badges / ranks) automatically be removed and added based on there post count etc.

So someone just joins they have newbie rank. As soon as they make 25 posts there newbie rank is removed and is the visitor role gets applied.

So esentially you set the newbie role to be removed when they reach 25 posts and set visitor role to be added instead.

So on and so forth. 50 posts removes visitor role and observer gets added instead.

I have roles setup that would have to be manually edited per user request to be updated. Having this automated would be amazing.

This idea is not limited to just forum posts and can extend to any newer features that might exist like reputation.

Thanks For Reading,
Rawr

**Comment by Derkades** [comment]
See also https://github.com/NamelessMC/Nameless/discussions/2147
